packages/runtime/src/helpers/edge.ts

```diff
@@ -442,6 +453,7 @@ export const writeEdgeFunctions = async ({
         pattern,
         // cache: "manual" is currently experimental, so we restrict it to sites that use experimental appDir
         cache: usesAppDir ? 'manual' : undefined,
+        generator: `${nextjsPluginVersion}`,
```

Comment:
This doesn't need to be a template string now and you can just use the variable.

```suggestion
      generator: nextjsPluginVersion`,
```